components/Tools/CrsConverter/ConversionButton.vue

```diff
@@ -35,15 +49,16 @@
   async function convert_files() {
     for (let i = 0; i < files.length; i++) {
       const params = {
-        geode_object: geode_object,
+        input_geode_object: input_geode_object,
         filename: files[i].name,
         input_crs_authority: input_crs["authority"],
         input_crs_code: input_crs["code"],
         input_crs_name: input_crs["name"],
```

Comment:
now that we are passing json as params, you could create an input_crs object with 3 properties (authority, code and name) if it is simpler to handle